### PR TITLE
Fix/0021514/5 3/background task rename dismiss to remove

### PR DESCRIPTION
**Pull request by d3r1w**
This PR proposes a fix for the bug described in Mantis: [0021514](https://www.ilias.de/mantis/view.php?id=21514)

### CODE REVIEW

File: lang/ilias_de.lang [hunk 1]
```diff
@@ -13789,7 +13789,7 @@ ecs#:#ecs_member_auth_type#:#Authentifizierungsmethode für Kursmitglieder
 background_tasks#:#background_tasks#:#Background Tasks
 background_tasks#:#background_tasks_running#:#Background Tasks
 background_tasks#:#scheduled#:#Geplant
-background_tasks#:#dismiss#:#Schliessen
+background_tasks#:#dismiss#:#Entfernen
```
Comment (chfsx) [inline lang/ilias_de.lang]: @d3r1w could you please rename the internal variable from `dismiss` to `remove` as well? thanks!

**Review by chfsx — CHANGES_REQUESTED** [review]
see my comment concerning the internal renaming, please

File: lang/ilias_de.lang (hunk 1 above)
Comment (d3r1w) [reply lang/ilias_de.lang]: @chfsx should I only change the langvar or also the task interface and the abstract task constants? According to 45c6fc580501c08e5162b03d11063090ef85f672.

**Comment by chfsx** [comment]
also the Constants and the Interface ( `getDismissOption` -> `getRemoveOption` )

File: lang/ilias_de.lang (hunk 1 above)
Comment (klees) [reply lang/ilias_de.lang]: Yay for consistency. Thx!

**Review by chfsx — APPROVED** [review]

**Pull request merged**